### PR TITLE
Addition of GDS client to the Docs dropdown menu in the header

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by davidoliverSP2** [comment]
Thanks @lidiazuin - I think the dropdown is also missing the new Query API, could we add it in this PR?
Check with @stefano-ottolenghi to see where in the list he thinks it should go!

File: src/partials/navbar-end.hbs [hunk 1]
```diff
@@ -93,6 +93,7 @@
               <span class="project-name">Neo4j Graph Data Science</span>
               <ul class="project-links">
                 <li><a href="/docs/graph-data-science/current/" class="project-link">Neo4j Graph Data Science Library</a></li>
+              <li><a href="/docs/graph-data-science-client/current/" class="project-link">Neo4j Graph Data Science Client</a></li>
```
Comment (stefano-ottolenghi) [inline src/partials/navbar-end.hbs:96]: Indent this as the previous `<li>` element.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -36,4 +36,4 @@ jobs:
 
       - name: Sync UI Bundle to S3
         run: |
-          aws s3 sync --follow-symlinks --include '*.zip' --no-progress build s3://${{ secrets.STATIC_CONTENT_BUCKET }}/build
+          aws s3 sync --follow-symlinks --include '*.zip' --no-progress build s3://${{ secrets.STATIC_CONTENT_BUCKET }}/build
```
Comment (stefano-ottolenghi) [inline .github/workflows/release.yml:39]: Remove changes to both workflow files from this PR, as they don't belong. It's good hygiene and will help any future troubleshooting involving those files.

**Review by MarkWoulfeNeo4j — APPROVED** [review]
LGTM

**Pull request merged**